notebooks/scikit-learn/20_newsgroups_automl.ipynb

```diff
@@ -0,0 +1,350 @@
+              "        text-align: right;\n",
+              "    }\n",
+              "</style>\n",
+              "<table border=\"1\" class=\"dataframe\">\n",
+              "  <thead>\n",
+              "    <tr style=\"text-align: right;\">\n",
+              "      <th></th>\n",
+              "      <th>text</th>\n",
+              "      <th>categories</th>\n",
+              "    </tr>\n",
+              "  </thead>\n",
+              "  <tbody>\n",
+              "    <tr>\n",
+              "      <th>16550</th>\n",
+              "      <td>From: will@futon.webo.dg.com (Will Taber) Subj...</td>\n",
+              "      <td>soc.religion.christian</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>13121</th>\n",
+              "      <td>From: zowie@daedalus.stanford.edu (Craig \"Powd...</td>\n",
+              "      <td>sci.space</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>15519</th>\n",
+              "      <td>From: boyle@cactus.org (Craig Boyle) Subject: ...</td>\n",
+              "      <td>rec.autos</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>10389</th>\n",
+              "      <td>From: erics@netcom.com (Eric Smith) Subject: R...</td>\n",
+              "      <td>rec.sport.baseball</td>\n",
+              "    </tr>\n",
+              "    <tr>\n",
+              "      <th>15554</th>\n",
+              "      <td>Organization: University of Illinois at Chicag...</td>\n",
+              "      <td>talk.politics.guns</td>\n",
+              "    </tr>\n",
+              "  </tbody>\n",
+              "</table>\n",
+              "</div>"
+            ],
+            "text/plain": [
+              "                                                    text  \\\n",
+              "16550  From: will@futon.webo.dg.com (Will Taber) Subj...   \n",
+              "13121  From: zowie@daedalus.stanford.edu (Craig \"Powd...   \n",
+              "15519  From: boyle@cactus.org (Craig Boyle) Subject: ...   \n",
+              "10389  From: erics@netcom.com (Eric Smith) Subject: R...   \n",
+              "15554  Organization: University of Illinois at Chicag...   \n",
+              "\n",
+              "                   categories  \n",
+              "16550  soc.religion.christian  \n",
+              "13121               sci.space  \n",
+              "15519               rec.autos  \n",
+              "10389      rec.sport.baseball  \n",
+              "15554      talk.politics.guns  "
+            ]
+          },
+          "metadata": {
+            "tags": []
+          },
+          "execution_count": 3
+        }
+      ]
+    },
+    {
+      "metadata": {
+        "id": "exFdr6wVz9co",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "## Export to CSV"
+      ]
+    },
+    {
+      "metadata": {
```

Comment:
Question: For AutoML do you need to move CSV file to a Cloud Storage bucket? If so will be nice to show how to do it.